taxonomies/labels.txt

```diff
@@ -19356,6 +19356,12 @@ nl:Kippenwelzijn
 fr:Lait de paturage
 nl:Weidemelk
 
+<en:Animal welfare
+fr:120 jours de pâturage
```

Comment:
Could you also add the English translations? it's easier if we always have an English name for all (or most) entries